src/content/docs/style-guide/writing-docs/article-templates/intro-branch-install-doc.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Get started with the branched install component
+metaDescription: 
+redirects: 
+---
+
+import styleGuideSimpleYamlFile from 'images/style-guide_diagram_simple-yaml-file.png'
+
+import styleGuideComplexYamlFile from 'images/style-guide_diagram_complex-yaml-file.png'
+
+Our branched install component for the docs site allows our users to select one or more options, which then displays content based on those choices. 
+
+## Overview of creation process [#overview]
+
+Before you start working with the branch install component, some things to consider: 
+
+* Ensure you definitely want to use the branched install. Consider if your problem can be solved with another solution (for example, the [tab component](/docs/style-guide/structure/intro-to-components)). 
+* Browse [the branched install docs we've made](#uses) and see what's possible. Select one you'd like to clone to use as a template. 
+* Finalize the logical flow of what your options will be and what content will be displayed. Ideally you want to be near-final in your content plans before creating a branch. For this reason, you might break the work into two or more jiras. 
+
+You can build the branched install component in many ways, and it may take you a good amount of effort to get it working, especially the more complex your structure is. But, that said, here's a high level view of what a creation process will often look like: 
```

Comment:
```suggestion
You can build the branched install component in many ways, and it may take you a good amount of effort to get it working, especially the more complex your structure is. But, that said, here's a high-level view of what a creation process will often look like: 
```